src/inapp/inapp.ts

```diff
@@ -481,6 +481,35 @@ export function getInAppMessages(
               clickedHostname === global.location.host || !clickedHostname;
             const { handleLinks } = payload;
 
+            /* 
+              if the _handleLinks_ option is set, we need to open links 
+              according to that enum. So the way this works is:
+
+              1. If _open-all-same-tab, then open every link in the same tab
+              2. If _open-all-new-tab, open all in new tab
+              3. If _external-new-tab_, open internal links in same tab, otherwise new tab.
+
+              This was a fix to account for the fact that Bee editor templates force
+              target="_blank" on all links, so we gave this option as an escape hatch for that.
+            */
+            const manageHandleLinks = (
+              sameTabAction: () => void,
+              newTabAction: () => void
+            ) => {
+              if (typeof handleLinks === 'string') {
+                if (
+                  handleLinks === 'open-all-same-tab' ||
+                  (isInternalLink && handleLinks === 'external-new-tab')
+                ) {
+                  sameTabAction();
+                } else {
+                  newTabAction();
+                }
+              } else if (link.getAttribute('target') === null) {
```

Comment:
getAttribute returns null rather than undefined when absent, this is the fix for the conditional logic

Reply:
also moves this below the handleLinks if statement, because handleLinks is supposed to override target values